addon/mixins/translate3d-mixin.js

```diff
@@ -73,22 +73,36 @@ export default Mixin.create({
    */
   willDestroyElement() {
     this._super(...arguments);
+  
+    let config = getOwner(this).resolveRegistration('config:environment');
+
+    let containerClone = this.element.parentElement.cloneNode(true);
```

Comment:
Use `parentNode` instead, as it is being used in other places and to increase compatibility a little bit.